contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -85,6 +85,8 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
   error PaymentTooLarge();
   error NoTransmittersSet();
   error InvalidCalldata();
+  event InternalError(bytes32 indexed requestId, string reason);
```

Comment:
Who will be consuming these new events?
Shall o11y/alerting ignore these?
Do these complement or replace RequestEnd event?